crowdsourcing/viewsets/task.py

```diff
@@ -227,6 +236,13 @@ def submit_results(self, request, *args, **kwargs):
             task_worker.task_status = task_status
             task_worker.save()
 
+            try:
+                user_preferences, created = models.UserPreferences.objects.get_or_create(user=request.user)
+                user_preferences.auto_accept = auto_accept
+                user_preferences.save()
```

Comment:
let's change this only when we need to, meaning only when the checkbox state is changed